core/store/migrate/migrations/0188_create_automation_upkeep_state.sql

```diff
@@ -0,0 +1,22 @@
+-- +goose Up
+
+CREATE TABLE evm_upkeep_states (
+  id SERIAL PRIMARY KEY,
+  work_id VARCHAR(32) NOT NULL,
```

Comment:
This is only 16 bytes. Again, are we _sure_ that string is the true underlying type?

Reply:
I'll switch all internals of the state store to `common.Hash` to ensure correctness in value types.